packages/blade/src/components/ActionList/styles/getBaseActionListItemStyles.ts

```diff
@@ -25,7 +26,7 @@ const getBaseActionListItemStyles = (
     borderColor: 'transparent',
     textAlign: isReactNative() ? undefined : 'left',
     backgroundColor: 'transparent',
-    padding: makeSize(props.theme.spacing[2]),
+    padding: makeSize(props.isMobile ? props.theme.spacing[3] : props.theme.spacing[2]),
```

Comment:
we can just use media query here rather than js check no?

Reply:
> we can just use media query here rather than js check no?

Overall isMobile provides a better ergonimics, with media query I need to specify screen width and all that which is already handled by the useBreakpoint hook which i can just pass onto this component.

Reply:
onMobile as prop to a component doesn't make much sense tbh. Sounds like it's an event. 

on tokens it makes sense that "onMobile screens our surface is going to be red"

<StyledActionListItem onMobile={} \/>